common/src/main/java/apoc/util/Util.java

```diff
@@ -1361,4 +1362,11 @@ public static <T> T withBackOffRetries(Supplier<T> func, long initialTimeout, lo
         }
         return result;
     }
+
+    // Get the current supported query language versions, if this list changes
+    // this function will error, on error please update!
+    public static List<String> getCypherVersions() {
+        assert QueryLanguage.ALL.size() == 2;
+        return List.of("5", "25");
+    }
```

Comment:
Could also be a compilation failure:

```suggestion
    public static List<String> getCypherVersions() {
        return QueryLanguage.ALL.stream()
          .map(l -> switch(l) {
            case Cypher5 -> "5";
            case Cypher25 -> "25";
          })
          .toList();
    }
```